apps/dashboard/src/trpc/routers/assistant.ts

```diff
@@ -0,0 +1,14 @@
+import { getLatestChat } from "@/actions/ai/storage";
+import { protectedProcedure } from "../init";
+import { createTRPCRouter } from "../init";
+
+export const assistantRouter = createTRPCRouter({
+  history: protectedProcedure.query(async ({ ctx: { teamId, session } }) => {
+    const data = await getLatestChat({
+      teamId: teamId!,
+      userId: session.user.id,
+    });
+
+    return data ?? null;
+  }),
+});
```

Comment:
I think it is possible for you to:

```suggestion
import type { TRPCRouterRecord } from "@trpc/server";

export const assistantRouter = {
  history: protectedProcedure.query(async ({ ctx: { teamId, session } }) => {
    const data = await getLatestChat({
      teamId: teamId!,
      userId: session.user.id,
    });

    return data ?? null;
  }),
} satisfies TRPCRouterRecord;
```
(applies for all other routers)
https://trpc.io/blog/announcing-trpc-v11#shorthand-router-definitions

I think it makes the type definitions faster as well.